### DISCUSSION TITLE
macOS damaged app

### DISCUSSION THREAD

**Discussion by mpcami8** [stakira/OpenUtau#1187] [Q&A]
Hello,
I hope whoever is reading this is doing well. When I tried opening the app it said, "'OpenUtau' is damaged and can't be opened. You should move it to the Trash". I tried the suggested commands to fix this problem but they did not work. Is there any way to fix this?

**Comment by BagelHero** [comment]
Hiya, hope your day is going okay too :)

Just to be clear, the suggested command found here?
https://github.com/stakira/OpenUtau/wiki/FAQ#on-macos-when-i-try-to-run-openutau-it-says-this-app-is-damaged

There's also definitely no error when running it?

**Comment by javanasse** [comment]
I also ran into the same issue. When running ```/usr/bin/xattr -rc /Applications/OpenUTAU.app``` nothing happens, no windows open, and no error in the shell. Any suggestions? Should I try building from source?

**Reply by BagelHero** [reply]
Be sure your Mac system settings (under privacy & security) has "Allow apps downloaded from" set to "Anywhere". I will note, though, it should be `xattr -rc /Applications/OpenUtau.app`, rather than `/usr/bin/xattr -rc /Applications/OpenUTAU.app` as far as I'm aware. You can also always right click and allow a one-time launch.
